src/app/shared/mydspace-actions/pdf-attestation-action/pdf-attestation-action.component.html

```diff
@@ -1,5 +1,6 @@
 <button *ngIf="isAuthorized$ | async"
         (click)="generateAttestationDownloadURL()"
-        class="btn btn-info mt-1 mb-3 ml-1" >
+        class="btn btn-info"
+        [ngClass]="additionalClasses">
```

Comment:
you should simple used 
```
class="btn btn-info {{ additionalClasses.join(' ') }}"
```

info from the web :

> When you do something like [ngClass]="sizingMethod()", especially without ChangeDetectionStrategy.OnPush, the method gets invoked every time Angular detect changes, so it can have poor performance.

Reply:
good to know !